remo/base/forms.py

```diff
@@ -1,13 +1,11 @@
-import happyforms
```

Comment:
Are you just removing happyforms because you don't care about it's form stripping functionality or do you have another way to do this now?